src/html/Quoting.cc

```diff
@@ -14,8 +14,7 @@
 #include <unordered_map>
 
 /*
- *  html_do_quote - Returns a static buffer containing the quoted
- *  string.
+ *  html_do_quote - Returns a static buffer containing the quoted string.
```

Comment:
This description belongs to the header file, where it should use a single-line Doxygen format. For example:

```
/// a static buffer containing an HTML-encoded version of the given c-string
```

Adjust as needed, of course. It would be nice to reference HTML encoding standard that we are implementing here, but I do not know whether it exists.